src/ui/components/NetworkMonitor/utils.ts

```diff
@@ -30,42 +30,74 @@ export type RequestSummary = {
   start: number;
   status: number;
   time: number;
+  type: RequestType;
   url: string;
 };
 
-export const REQUEST_TYPES = {
+export const CANONICAL_REQUEST_TYPES = [
+  "CSS",
+  "Fetch/XHR",
+  "Font",
+  "HTML",
+  "Image",
+  "Javascript",
+  "Manifest",
+  "Media",
+  "Other",
+  "WASM",
+  "Websocket",
+] as const;
+
+export type CanonicalRequestType = typeof CANONICAL_REQUEST_TYPES[number];
```

Comment:
To elaborate, I usually try to avoid UI strings being primary keys in object. This is also a slightly better fit for an enum because you can reuse the enum keys. Something for instance like
```
export enum CanonicalRequestType {
  CSS,
  XHR,
  FONT,
  HTML,
  IMAGE,
  JS,
  MANIFEST,
  MEDIA,
  OTHER,
  WASM,
  WS,
}

// I'm not exporting this anymore because `CanonicalRequestType` is usable for the dropdown.
const REQUEST_TYPES: Record<string, CanonicalRequestType> = {
  document: CanonicalRequestType.HTML,
  subdocument: CanonicalRequestType.HTML,
  objectSubdoc: CanonicalRequestType.HTML,

  // and so on
};

export const REQUEST_TYPE_INFO: Record<CanonicalRequestType, { label: string, icon: string }> = {
  [CanonicalRequestType.HTML]: { icon: "description", string: "HTML" },
  [CanonicalRequestType.JS]: { icon: "code", string: "JavaScript" },

  // and so on
};
```